console/src/main/java/org/nervos/ckb/TransactionClient.java

```diff
@@ -26,11 +28,18 @@ public static void main(String[] args) throws Exception {
                 "ckt1qyq9ngn77wagfurp29738apv738dqgrpqpssfhr0l6", new BigInteger("12000000000")),
             new Receiver(
                 "ckt1qyq2pmuxkr0xwx8kp3ya2juryrygf27dregs44skek", new BigInteger("15000000000")));
+    System.out.println("Sender's balance: " + getBalance(senderPrivateKey).toString(10));
```

Comment:
We should consider remove this `console` package. The example here is already simple enough to just appear on doc.

Reply:
`console` package can generate all modules to one jar file, not just an example package. 
So I suggest we can rename `TransactionClient` to `TransactionExample` .